taxi/forms.py

```diff
@@ -50,3 +50,41 @@ def validate_license_number(
         raise ValidationError("Last 5 characters should be digits")
 
     return license_number
+
+
+class DriverSearchForm(forms.Form):
+    username = forms.CharField(
+        max_length=255,
+        required=False,
+        label="",
+        widget=forms.TextInput(
+            attrs={"placeholder": "Search by username"}
+        )
+    )
+
+
+class CarSearchForm(forms.ModelForm):
```

Comment:
The CarSearchForm should inherit from forms.Form instead of forms.ModelForm because it is used for searching, not for creating or updating model instances.

Reply:
+